src/main/java/com/example/pfairplayservice/model/Member.java

```diff
@@ -44,18 +54,36 @@
     @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
     private Date recentLoginDate;
 
-    public static Member from(MemberEntity member) {
+    public static Member from(MemberEntity memberEntity) {
 
-        Member result = new Member(member.getUid(), member.getName(), member.getId(), member.getPassword()
-                , member.getBirthday(), member.getAddress(), member.getPhoneNumber(), Position.from(member.getPreferPosition())
-                , member.getLevel(), member.getPhoneNumberDisclosureOption(), member.getJoinDate(), member.getRecentLoginDate());
+        Member result = new Member(memberEntity.getUid(), memberEntity.getName(), memberEntity.getId(), Team.fromList(memberEntity.getTeamEntityList()), memberEntity.getPassword()
+                , memberEntity.getBirthday(), memberEntity.getAddress(), memberEntity.getPhoneNumber(), Position.from(memberEntity.getPreferPosition())
+                , memberEntity.getLevel(), memberEntity.getPhoneNumberDisclosureOption(), memberEntity.getJoinDate(), memberEntity.getRecentLoginDate());
         return result;
     }
 
     public static MemberEntity to(Member member) {
-        MemberEntity result = new MemberEntity(member.getUid(), member.getName(), member.getId(), member.getPassword()
-                , member.getBirthday(), member.getAddress(), member.getPhoneNumber(), member.getPreferPosition().getPosition()
-                , member.getLevel(), member.getPhoneNumberDisclosureOption(), member.getJoinDate(), member.getRecentLoginDate());
+        MemberEntity result = new MemberEntity(member.getUid(),Team.toList(member.getTeamList()),member.getName(),member.getId()
+            ,member.getPassword(),member.getBirthday(),member.getAddress(),member.getPhoneNumber(),member.getPreferPosition().getPosition()
+            ,member.getLevel(),member.getPhoneNumberDisclosureOption(),member.getJoinDate(),member.getRecentLoginDate());
         return result;
     }
+
+    public static List<MemberEntity> toList(List<Member> memberList) {
+        if (memberList == null) return null;
+        List<MemberEntity> memberEntityListTemp = null;
+        memberList
+                .stream()
+                .forEach(member -> memberEntityListTemp.add(Member.to(member)));
+        return memberEntityListTemp;
+    }
+
+    public static List<Member> fromList(List<MemberEntity> memberEntityList) {
+        if (memberEntityList == null) return null;
+        List<Member> memberListTemp = null;
+        memberEntityList
+                .stream()
+                .forEach(memberEntity -> memberListTemp.add(Member.from(memberEntity)));
+        return memberListTemp;
+    }
```

Comment:
I think this methods are not necessary for this class.
This transformation could be in the caller context. It can be performed by one stream function as you wrote.

Reply:
I think toList, fromList are useful when converting List<Object> and List<ObjectEntity> from one another.

Without them, code would be complicated.

ex)

`

        Member result = Member.builder()
                .uid(memberEntity.getUid())
                .id(memberEntity.getId())
                .password(memberEntity.getPassword())
                .name(memberEntity.getName())
                .teamList(memberEntity.getTeamEntityList())  // I think this line would be complicated
                .birthday(memberEntity.getBirthday())
                .address(memberEntity.getAddress())
                .phoneNumber(memberEntity.getPhoneNumber())
                .preferPosition(Position.from(memberEntity.getPreferPosition()))
                .level(memberEntity.getLevel())
                .phoneNumberDisclosureOption(memberEntity.getPhoneNumberDisclosureOption())
                .joinDate(memberEntity.getJoinDate())
                .recentLoginDate(memberEntity.getRecentLoginDate())
                .build();

        return result;
    }`

Reply:
Why you write builder code for example?

```java
MemberEntity result = MemberEntity.builder()
                // blah
                .teamEntityList(member.getTeamList().stream().map(it-> it.toTeamEntity()).collect(Collections.toList()))
                // blah
                .build();

        return result;
```